R/database.R

```diff
@@ -865,6 +925,13 @@ add_icgc_metadata = function(incoming_metadata){
   icgc_publ = mutate(icgc_publ, sex = str_to_upper(sex))
 
   icgc_raw_path = paste0(repo_base,"data/metadata/raw_metadata/ICGC_MALY_seq_md.tsv")
+
```

Comment:
The files in this function are mostly in the GAMBL repo. Please refer to my earlier comment and revise these messages accordingly